spell-check/action.yaml

```diff
@@ -18,6 +18,10 @@ inputs:
     description: ""
     required: false
     default: ${{ github.token }}
+  incremental-files-only:
```

Comment:
Could you update the README as well? :pray:

Reply:
Oh, I forgot!
I'll do :+1:

Reply:
Done!